From 0b7e68e053cc06c65e9072d4d07bc6fb62eacee3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Fri, 25 Oct 2024 14:35:21 +0200
Subject: [PATCH 1336/1695] dt-bindings: stm32_rng: add RNG support for
 STM32MP21
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

update bindings to support rng in STM32MP21.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 0386fe63b1c4f7bc702f853917de60afc287e228 dt-bindings: stm32_rng: add RNG support for STM32MP21]
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Change-Id: I2d7a4753fda39c84e548a9bd83f25e83655fca30
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/414254
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 documentation/devicetree/bindings/rng/st,stm32-rng.yaml | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/documentation/devicetree/bindings/rng/st,stm32-rng.yaml b/documentation/devicetree/bindings/rng/st,stm32-rng.yaml
index 0ad23b9fb..8711107f9 100644
--- a/documentation/devicetree/bindings/rng/st,stm32-rng.yaml
+++ b/documentation/devicetree/bindings/rng/st,stm32-rng.yaml
@@ -18,6 +18,7 @@ properties:
     enum:
       - st,stm32-rng
       - st,stm32mp13-rng
+      - st,stm32mp21-rng
       - st,stm32mp25-rng
 
   reg:
@@ -57,7 +58,9 @@ allOf:
       properties:
         compatible:
           contains:
-            const: st,stm32mp25-rng
+            enum:
+              - st,stm32mp21-rng
+              - st,stm32mp25-rng
     then:
       properties:
         clocks:
-- 
2.47.2

